### PR TITLE
Switch from Aktiv Grotesk to Inter

### PR DESCRIPTION
**Pull request by connor-baer**
Addresses ticket [DSYS-707](https://sumupteam.atlassian.net/browse/DSYS-707). Relates to https://github.com/sumup-oss/circuit-ui/pull/2653.

## Purpose

SumUp switched from Museo Sans to Aktiv Grotesk as its brand font six years ago. The recent switch to the black&white color scheme has shifted more responsibility on the typography to provide hierarchy and visual interest. However, only regular and bold font weights are loaded on the web (the mobile apps use native fonts). Loading additional font weights and font styles would significantly impact page load speeds and, in turn, conversion rates.

[Variable fonts](https://fonts.google.com/knowledge/glossary/variable_fonts) offer a modern solution by combining a continuous range of weights and other "axes" into a reasonably small file. A variable font version of Aktiv Grotesk exists; however, its commercial license is prohibitively expensive ([DSYS-679](https://sumupteam.atlassian.net/browse/DSYS-679)).

We've identified Inter as a replacement. Inter is available as a variable font, freely usable in commercial projects and an almost perfect match to Aktiv Grotesk, especially with an adjusted negative letter spacing.

## Approach and changes

- Generate a new `fonts.css` file containing the font face declarations for Inter regular & italic. The file is exported from `@sumup-oss/design-tokens` and the font files are hosted on static.sumup.com
- Add documentation for loading fonts. Next.js-based projects should use [`next/font`](https://nextjs.org/docs/app/building-your-application/optimizing/fonts) to self-host the fonts

## Definition of done

* [x] Development completed
* [x] Reviewers assigned
* [x] Unit and integration tests
* [x] Meets minimum browser support
* [x] Meets accessibility requirements

### CODE REVIEW

File: packages/design-tokens/themes/fonts.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  /* fallback */
+  {
+    'font-family': 'Inter-Fallback',
+    'src': 'local("Arial")',
+    'ascent-override': '90.49%',
+    'descent-override': '22.56%',
+    'line-gap-override': '0%',
+    'size-adjust': '107.06%',
+  },
+] satisfies FontFace[];
```
Comment (nicosommi) [inline packages/design-tokens/themes/fonts.ts:156]: Typescript ✨ TIL `satisfies`

File: templates/nextjs/template/app/layout.tsx [hunk 2]
```diff
@@ -1,9 +1,17 @@
 import type { Metadata, Viewport } from 'next';
+import { Inter } from 'next/font/google';
 
 import '@sumup-oss/design-tokens/light.css';
 import '@sumup-oss/circuit-ui/styles.css';
 
-import { PreloadResources } from './preload-resources';
+const inter = Inter({
+  subsets: ['latin'],
+  // FIXME: Re-enable once https://github.com/vercel/next.js/issues/68395 has been resolved
```
Comment (nicosommi) [inline templates/nextjs/template/app/layout.tsx:9]: Should this phrasing be for the template developer or the developer user? 🤔 
Or maybe mention to the later, "You can uncomment this line it if you have Next.js > 14.x.y" ?

File: packages/design-tokens/README.md [hunk 3]
```diff
@@ -37,6 +38,56 @@ The application code must be processed by a bundler that can handle CSS files. [
...
+  href="https://static.sumup.com/fonts/Inter/Inter-normal-latin.woff2"
+  as="font"
+  type="font/woff2"
+  crossorigin
+/>
+```
+
+#### Next.js
+
+If you're using Next.js 13+, use the built-in [font optimization](https://nextjs.org/docs/app/building-your-application/optimizing/fonts) feature with this recommended configuration instead:
```
Comment (nicosommi) [inline packages/design-tokens/README.md:65]: 👏  I like that you can leverage existing tools without restrictions

**Review by nicosommi — APPROVED** [review]
Great job decoupling the font declarations from the theme, that gives plenty of flexibility.

File: templates/nextjs/template/app/layout.tsx (hunk 2 above)
Comment (connor-baer) [reply templates/nextjs/template/app/layout.tsx:9]: It's a note to myself (the template developer). The PR has already been merged; I'm just waiting for a new version to be released.

**Pull request merged**